20180619/README.md

```diff
@@ -0,0 +1,359 @@
+# Quick and easy way to start with Matchmore: React-native
+
+We already showed you how to do it in Swift iOS, now it is time for the new kid on the block: _React-Native_.
+After this you will have a good understanding how to make your own Dating App with Matchmore!
+
+Before starting, it will be beneficial to have some React Native know-how before. And we will use Yarn!
+
+Lets start of with creating a React-Native app
+
+```
+react-native init DatingApp
```

Comment:
how do I get `react-native` ?

Reply:
Not part of this blog post, your should know react native before reading this.

Reply:
👌